### PR TITLE
Fix #6622 via CSS (multiline on verify page)

### LINKED ISSUE #6622: Preview of new nuget package. Incorrect display of multiline description and release note
Preview page displays a multiline text (description and release note of new package) in single line.

The page of published package displays a multiline text without problems.

It is creates some inconvenience.

See screenshots.
2018_11_04--nuget-01-preview_page
2018_11_04--nuget-02-package_page

### PR DESCRIPTION
**Pull request by robertmuehsig**
As suggested in the issue, this should solve the bug. Not sure if I found the correct location for this less changes. 

Summary of the changes (in less than 80 characters):

* Added a new less rule to display multiline in a correct way
* The CSS is should only affect the verify package site.
* Didn't found any other CSS/LESS stuff, so I used the "common-edit-metadata.less" file. 

Works in more or less all modern browsers :)

Addresses https://github.com/NuGet/NuGetGallery/issues/6622

### CODE REVIEW

**Comment by joelverhagen** [comment]
This is a different implementation that the display package page due to complexities mentioned by @scottbommarito in the original issue. Won't `pre-wrap` show consecutive spaces? Wouldn't `pre-line` be a bit closer to the display package behavior?

I am kind of curious how tab characters are displayed... in both cases 😄 

Let's just make sure we're picking the most appropriate `white-space` value then we can :shipit:.

**Comment by robertmuehsig** [comment]
Ok, I tried something. I changed the description from the original package and uploaded it to my dev environment:

Screenshot from the display package page:

image

The content was changed like this:

image

Now let's see how `pre-wrap` renders the page:

image

Now with `pre-line`:

image

To compare - this would be rendered without the PR:

image

I think `pre-line` is the better choice here - at least it is more consistent with the display package page.

File: src/Bootstrap/less/theme/common-edit-metadata.less [hunk 1]
```diff
@@ -55,4 +55,12 @@
         max-width: 300px;
         max-height: 300px;
     }
+}
+
+#verify-package-container {
+    .verify-package-field { 
+	    p {
+            white-space: pre-line;
+        }
```
[the remark this answers is not in the document]
Comment (robertmuehsig) [reply src/Bootstrap/less/theme/common-edit-metadata.less]: I hope my last commit fixed this. The resulting css is still the same.

**Comment by joelverhagen** [comment]
Great analysis, @robertmuehsig!

**Comment by joelverhagen** [comment]
Changed the base branch to `dev`.

**Comment by joelverhagen** [comment]
Looks like this still has a different behavior for indentation done with space characters. We can't win all the battles I think with just CSS so this is just fine for now.

**Pull request merged**